deps.ts

```diff
@@ -25,6 +25,7 @@ export {
   createReporter,
   type Reporter as GaReporter,
 } from "https://deno.land/x/g_a@0.1.2/mod.ts";
+export { GA4Report } from "https://raw.githubusercontent.com/denoland/ga4/main/mod.ts";
```

Comment:
We should wait until this library is officialy released before landing this PR

Reply:
alright

Reply:
leaving this convo unresolved until the google analytics 4 experiments are done your side :)